android/app/src/main/java/com/sample/manifest/Manifest.kt

```diff
@@ -2,14 +2,20 @@ package com.sample.manifest
 
 import com.squareup.moshi.JsonClass
 
-typealias Components = Map<String, Component>
+typealias Components = Array<Component>
```

Comment:
The benefits of having a type alias are not that significant with removal of the `Map`. Thus, I recommend to remove it. Also, it is not common to use `Array` as it doesn't implement an immutable collection API. I would replace with `List<T>` instead.